runtime/util/hshelp.c

```diff
@@ -482,6 +485,173 @@ areSingleSlotConstantRefsIdentical(J9ROMConstantPoolItem * romCP1, U_32 index1,
 	return utfsAreIdentical(J9ROMSTRINGREF_UTF8DATA((J9ROMStringRef *) ref1), J9ROMSTRINGREF_UTF8DATA((J9ROMStringRef *) ref2));
 }
 
+/*
+ * Helper methods for areCallSiteDataMethodsEquivalent
+ */
+
+/* ROM call site data header size (16 bits of method handle address + additional argument count) */
+static const U_16 callSiteHeaderOffset = 2;
+
+static UDATA
+isMethodHandleAField(UDATA cpType) {
+	return (cpType == MH_REF_GETFIELD) || (cpType == MH_REF_GETSTATIC) || (cpType == MH_REF_PUTSTATIC) || (cpType == MH_REF_PUTFIELD);
```

Comment:
I tend to use switch statements for this.  Not necessary to change, but something to think about for similar code:

```c
switch(cpType) {
case MH_REF_GETFIELD:
case MH_REF_GETSTATIC:
case MH_REF_PUTSTATIC:
case MH_REF_PUTFIELD:
    return true;
}
return false;
}
```